rio_tiler/io/stac.py

```diff
@@ -76,7 +76,7 @@ def _get_assets(
 
 
 @attr.s
-class STACReader(MultiBaseReader):
+class STACReader(MultiBaseReader, SpatialMixin):
```

Comment:
Is this necessary? The `MultiBaseReader` inherits from `BaseReader`, which already contains the `SpatialMixin`, right?